### ISSUE TITLE
Possible to use multiple background layers?

### ISSUE THREAD

**Issue by ctbucha** [SentiaAnalytics/bs-css#79]
I'm trying to an image url on top of a linear gradient as a background, but it seems like the type system is limiting me to one or the other. Is there a way to use multiple background layers with bs-css?

E.g. generate something like this:
```
background: url("../img/bg-pattern.png"), linear-gradient(to left, #7b4397, #dc2430);
```

**Comment by ctbucha** [comment]
FYI, my current workaround is to use a nested `<div>` tag with the other background layer. 

```
module Styles = { 
  open Css;

  let masthead = 
    style([
      background(
        linearGradient(
          deg(90),
          [(0, rgb(0, 0, 255)), (100, rgb(255, 0, 0))],
        ),  
      ),
      height(vh(100.0)),
    ]);
  let masthead2 = 
    style([background(url(backgroundUrl)), height(pct(100.0))]);
};
...
<div className=Styles.masthead>
  <div className=Styles.masthead2>
  </div>
</div>
```

**Comment by cullophid** [comment]
It's not currently part of the API, but pull requests are welcome..
Something like ```
backgrounds ([url("blabla"), black])
```

**Comment by giraud** [comment]
added a PR that implements a backgrounds

**Comment by giraud** [comment]
fixed in 7.3.0
